### PR TITLE
Test configureFirebase() and configureSentry()

### PR DESCRIPTION
**Pull request by knickmack**

### CODE REVIEW

File: config/__tests__/configureSentry.test.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  it("should find the org environment variable", () => {
+    expect(SENTRY_ORG).toBeTruthy();
+  });
+
+  it("should find the project environment variable", () => {
+    expect(SENTRY_PROJECT).toBeTruthy();
+  });
+
+  it("should initialize Sentry", () => {
+    const spy = jest.spyOn(SentryExpo, "init").mockImplementationOnce(() => {});
```
Comment (knickmack) [inline config/__tests__/configureSentry.test.ts:29]: Curses, I think we have 100% test coverage if I can figure out how to spy on the `Sentry.setRelease()`

File: config/configureFirebase.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import firebase from "firebase";
+import * as firebase from "firebase/app";
```
Comment (knickmack) [inline config/configureFirebase.ts:1]: Changed these, as this is how i needed to import them for `jest.spyOn` to work, and frankly, this is what's in the docs, so i don't know why i had it the other way to begin with 😆

File: config/__tests__/configureSentry.test.ts (hunk 1 above)
Comment (knickmack) [reply config/__tests__/configureSentry.test.ts:29]: To difficult, not worth the effort 💩

Comment (ltphan) [reply config/__tests__/configureSentry.test.ts:29]: These are good tests to follow for other configurations into our application when we do authorization and authentication!

**Pull request merged**